src/lib/annotations/drawing/DrawingPath.js

```diff
@@ -0,0 +1,101 @@
+class DrawingPath {
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+    /** @property {Array} - The array of coordinates that form the path */
+    path;
+
+    /** @property {number} - The maximum X position of all coordinates */
+    maxX;
+
+    /** @property {number} - The maximum Y position of all coordinates */
+    maxY;
+
+    /** @property {number} - The minimum X position of all coordinates */
+    minX;
+
+    /** @property {number} - The minimum Y position of all coordinates */
+    minY;
+
+    /**
+     * [constructor]
+     * @return {DrawingPath} DrawingPath instance
+     */
+    constructor() {
+        this.path = [];
+        this.maxX = -Infinity;
+        this.maxY = -Infinity;
+        this.minX = Infinity;
+        this.minY = Infinity;
+    }
+
+    /**
+     * Add position to coordinates and update the bounding box
+     *
+     * @param {number} xPos xPosition to be part of the drawing
+     * @param {number} yPos yPosition to be part of the drawing
+     * @return {void}
+     */
+    addCoordinate(xPos, yPos) {
+        if (xPos < this.minX) {
```

Comment:
AFAIK none of these ifs will ever take, as nothing can be greater or less than Infinity/-Infinity